docs/semgrep-supply-chain/license-compliance.md

```diff
@@ -135,4 +135,22 @@ Exempted packages appear in the Supply Chain > **Settings** tab.
 
 Package exemptions are currently version-specific. Each version used must be exempted individually.
 
+### Custom dependency exceptions
+
+Custom dependency exceptions allow you to prevent Semgrep from blocking a pull
+request or merge request due to licensing issues. For example, if
+`bitwarden/cli@2023.9.0`, which has a GPL-3.0 license, is on the allowlist,
+setting a custom dependency exception means that the exclusion won't fail when
+upgrading to `bitwarden/cli@2023.9.1`.
+
+To set a custom dependency exception:
+
+1. Log into Semgrep Cloud Platform and navigate to **Supply Chain** > <i class="fa-solid fa-gear"></i> **Settings**.
+2. Under **Custom Dependency Exceptions**, click **Add custom exception**.
```

Comment:
The style guide recommends not to use location-specific terms: https://developers.google.com/style/word-list#under

Very minor, so feel free to break the rule on this one. But do recall our UI tends to change, so what's "under" now may be "next to" in another version.